examples/tts/conf/audio_codec/encodec.yaml

```diff
@@ -90,7 +90,7 @@ model:
       - _target_: nemo.collections.tts.parts.utils.callbacks.AudioCodecArtifactGenerator
         log_audio: true
         log_encoding: true
-        log_quantized: true
+        log_dequantized: true
```

Comment:
The final codebook embeddings are a quantized version of the encoder output, so calling them "dequantized" might be misleading. I would favor keeping the convention in EnCodec and DAC and referring to the codebook indices as "codes" (instead of just "indices") and the corresponding embeddings as "quantized".

Reply:
After talking about it, I agree it makes the most sense to refer to the indices as "tokens" and the codebook embeddings can either be called "codes" or "dequantized" depending on the context. Some of the renaming and convention changes can be left for a future PR.